core/kazoo_tasks/Makefile

```diff
@@ -3,4 +3,12 @@ PROJECT = kazoo_tasks
 
 all: compile
 
+test: check_csvs
+
+check_csvs:
+	@stat -t -- test/*.csv >/dev/null 2>/dev/null ; if [[ $$? > 0 ]] ; then echo WARNING: run 'make csvs' for more test data ; fi
```

Comment:
hm maybe there can be some licensing issues? but if they're not too big I'd rather we include them in kazoo_csv/test or kazoo_csv/priv/fixtures